addons/point_of_sale/static/src/js/models.js

```diff
@@ -2971,6 +2980,26 @@ exports.Order = Backbone.Model.extend({
         }
         return round_pr(due, this.pos.currency.rounding);
     },
+    get_rounding_applied: function() {
+        if(this.pos.config.cash_rounding) {
+            var total = round_pr(this.get_total_with_tax(), this.pos.cash_rounding[0].rounding);
```

Comment:
Legally, you can only round if amount is >= 0.05, at least in Belgium. This might be something to consider.

Reply:
@nim-odoo no, what you say is actually wrong. ;) See https://economie.fgov.be/nl/themas/verkoop/prijsbeleid/betalingen/afronding-van-cashbetalingen/afronden-van-cashbetalingen-0 under the question "Hoe gebeurt de afronding van cashbetalingen?":
![image](https://user-images.githubusercontent.com/6352350/70040217-725d3480-15bb-11ea-9d8e-df19cc034eaa.png)

Reply:
@Yenthe666 You are wrong, you can read the law here: https://economie.fgov.be/nl/themas/verkoop/prijsbeleid/betalingen/afronden-van-cashbetalingen-de in chapter 'Hoe gaat het afronden in zijn werk?'

Reply:
@pimodoo well to quote:

> Het te betalen totaalbedrag dat eindigt op 1 of 2 cent, wordt afgerond naar het lagere x,x0.
Het te betalen totaalbedrag dat eindigt op 3, 4, 6 of 7 cent, wordt afgerond naar x,x5.
Het te betalen totaalbedrag dat eindigt op 8 of 9 cent, wordt afgerond naar het hogere x,(x+1)0.

This means you're rounding also if it is smaller than 5. You're rounding down instead of up in that case. Right? Or am I so wrong now? 😆

Reply:
@Yenthe666 Did you read the chapter I mentionned?

> het te betalen bedrag groter is dan 5 cent

Reply:
Aha, I missed that specific sentence. TIL, you guys are right!